pt-BR/mail/chrome/messenger/messengercompose/messengercompose.dtd

```diff
@@ -206,40 +206,40 @@
 <!ENTITY attachButton.label "Anexar">
 <!ENTITY spellingButton.label "Ortografia">
 <!ENTITY saveButton.label "Salvar">
-<!ENTITY tagButton.label "Assign Topic">
-<!ENTITY throbberItem.title "Activity Indicator">
-<!ENTITY signatureButton.label "Signatures">
-<!ENTITY compose.sidebar.label "Sidebar">
+<!ENTITY tagButton.label "Atribuir tópico">
+<!ENTITY throbberItem.title "Indicador de atividade">
+<!ENTITY signatureButton.label "Assinaturas">
+<!ENTITY compose.sidebar.label "Barra lateral">
 
 <!-- Mail Toolbar Tooltips -->
-<!ENTITY sendButton.tooltip "Enviar esta mensagem agora"> 
-<!ENTITY sendlaterButton.tooltip "Armazenar a mensagem na pasta “Saída”"> 
-<!ENTITY quoteButton.tooltip "Citar a mensagem anterior"> 
-<!ENTITY addressButton.tooltip "Selecionar destinatários no catálogo de endereços"> 
-<!ENTITY attachButton.tooltip "Anexar arquivos"> 
-<!ENTITY saveButton.tooltip "Salvar a mensagem na pasta “Rascunhos”"> 
+<!ENTITY sendButton.tooltip "Enviar esta mensagem agora">
+<!ENTITY sendlaterButton.tooltip "Armazenar a mensagem na pasta “Saída”">
+<!ENTITY quoteButton.tooltip "Citar a mensagem anterior">
+<!ENTITY addressButton.tooltip "Selecionar destinatários no catálogo de endereços">
+<!ENTITY attachButton.tooltip "Anexar arquivos">
+<!ENTITY saveButton.tooltip "Salvar a mensagem na pasta “Rascunhos”">
 <!ENTITY cutButton.tooltip              "Recortar">
 <!ENTITY copyButton.tooltip             "Copiar">
 <!ENTITY pasteButton.tooltip            "Colar">
-<!ENTITY tagButton.tooltip "Add topic(s) to message">
-<!ENTITY signatureButton.tooltip "Insert a signature">
-<!ENTITY compose.sidebar.tooltip "Open the sidebar">
+<!ENTITY tagButton.tooltip "Adicionar tópico(s) à mensagem">
+<!ENTITY signatureButton.tooltip "Inserir uma assinatura">
+<!ENTITY compose.sidebar.tooltip "Abrir a barra lateral">
 
 <!-- Headers -->
-<!ENTITY fromHeader.label "From">
+<!ENTITY fromHeader.label "De">
 <!ENTITY fromHeader.accesskey "r">
-<!ENTITY subjectHeader.label "Subject">
+<!ENTITY subjectHeader.label "Assunto">
 <!ENTITY subjectHeader.accesskey "S">
-<!ENTITY toAddr.label "Para:"> 
-<!ENTITY ccAddr.label "Cc:"> 
-<!ENTITY bccAddr.label "Cco:"> 
-<!ENTITY replyAddr.label "Responder a:"> 
-<!ENTITY newsgroupsAddr.label "Newsgroup:"> 
-<!ENTITY followupAddr.label "Followup-To:"> 
-<!ENTITY tagHeader.label "Topics">
+<!ENTITY toAddr.label "Para:">
+<!ENTITY ccAddr.label "Cc:">
+<!ENTITY bccAddr.label "Cco:">
```

Comment:
CORRECTIONS NEEDED it should be <!ENTITY bccAddr.label "Bcc:">
@brunotiussi please don't translate the string Bcc.